### DISCUSSION TITLE
error code -1007?

### DISCUSSION THREAD

**Discussion by oldshaghat** [MoleMan1024/audiowagon#84] [General]
Happily the combination of updated Audiowagon and updated Volvo software == I have music in the car again.  We were seeing that it was throwing a lot of error code notifications (iirc the number was -1007, but I maybe misremember) but the tracks would play fine in spite of it. 

Had a couple of tracks that were found / discovered and were non-playable with no error code (both of them cases where it was single tracks if that matters) - I need to double check that we have an mp3 and not an m4a or something that I put on the stick by mistake.

**Comment by MoleMan1024** [comment]
hi there. This error code is from media player which means [malformed data](https://developer.android.com/reference/android/media/MediaPlayer#MEDIA_ERROR_MALFORMED) 
The media player errors come from the underlying Android, and most of the time I also don't know what they mean. Based on error code, something might be wrong with the files.

**Reply by oldshaghat** [reply]
strange that the files still play then if they are malformed
